its/src/test/java/com/sonar/it/csharp/IncrementalAnalysisTest.java

```diff
@@ -22,6 +22,7 @@
 import com.sonar.it.shared.TestUtils;
 import com.sonar.orchestrator.build.BuildResult;
 import com.sonar.orchestrator.build.ScannerForMSBuild;
+
```

Comment:
There's something wrong with your IntelliJ setup. When I apply formatting, these empty lines get removed.

See https://github.com/SonarSource/sonar-developer-toolset#code-style-configuration-for-intellij